reference/library/src/morpheus-master/sass/modules/_toolmenu.scss

```diff
@@ -49,8 +49,8 @@ body.#{$namespace}toolMenu-collapsed{
 				}
 				&.#{$namespace}collapseTools{
 					i{
-					   @include rotate(180deg);
-						@include transform( rotate(180deg) );
+					   /* @include rotate(180deg); */
+						@include transform( scaleX(-1) ); /* flip horizontally */
```

Comment:
The change looks positive to me.

Can you just delete the line you comment out here and fix the indenting?